src/main/java/net/starlark/java/eval/cpu_profiler_windows.cc

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Starlark CPU profiler stubs for unsupported platforms.
```

Comment:
I thought we are not implementing the cpu profiler on Windows, what's up with this file?

Reply:
Added by mistake. I thought I left this out but I guess not